package.json

```diff
@@ -7,19 +7,46 @@
     "author": "CXL <leho@cxl.com>",
     "license": "MIT",
     "peerDependencies": {
-        "eslint": "^7.7.0",
-        "husky": "^4.2.5",
-        "lint-staged": "^10.2.13",
-        "prettier": "^2.1.1"
+        "@10up/eslint-config": "^2.1.1",
+        "@10up/stylelint-config": "^1.0.11",
+        "@wordpress/eslint-plugin": "^8.0.2",
+        "babel-eslint": "^10.1.0",
+        "eslint": "^7.17.0",
+        "eslint-config-airbnb": "^18.2.1",
+        "eslint-config-airbnb-base": "^14.2.1",
+        "eslint-config-prettier": "^7.1.0",
+        "eslint-plugin-import": "^2.22.1",
+        "eslint-plugin-jsdoc": "^31.6.0",
+        "eslint-plugin-jsx-a11y": "^6.4.1",
+        "eslint-plugin-prettier": "^3.3.1",
+        "eslint-plugin-react": "^7.22.0",
+        "eslint-plugin-react-hooks": "^4.2.0",
+        "husky": "^4.3.7",
```

Comment:
Ideally, avoid bumping existing stuff simultaneously so it'd be better to identify what new is.

Reply:
> Ideally, avoid bumping existing stuff simultaneously so it'd be better to identify what new is.

I will rebase the code and try to keep the original version for this PR.